R/plot-boxwhisker.R

```diff
@@ -62,8 +62,6 @@ plotBoxWhisker <- function(data,
   if (plotConfiguration$outliers) {
     plotObject <- .addOutliers(data, metaData, dataMapping, plotConfiguration, plotObject)
   }
-  plotObject <- setLegendPosition(plotObject)
-  plotObject <- setLegendFont(plotObject)
```

Comment:
This is not required anymore?
Particularly in the changes below I did not see anything regarding the Legend **Position**(?)